packages/skeleton-svelte/src/lib/components/Accordion/Accordion.test.svelte

```diff
@@ -0,0 +1,20 @@
+<!-- Accordion Test Component -->
+
+<script lang="ts">
+	import { Accordion } from '$lib/index.js';
+	let { childProps = {} } = $props();
+	let chilPropsSpread = childProps ?? {};
```

Comment:
Any reason why you keep creating these `xSpread` props? This won't be reactive so we should watch out for that aswell.

Reply:
Yeah I mean to bring this up before all the issues today - when you try to spread the fallback `{}` there's a type error with Svelte check. This was the only way I was able to work around it. I'm open to suggestions. Just revert the change and run `pnpm check` to see what I mean.

Reply:
Do you mind if I push the proper fix for this?

Reply:
Pushed a fix, checks pass now.